support/sier-codec/src/json.rs

```diff
@@ -0,0 +1,69 @@
+use crate::{Error, Object, StructDef, Type, Value};
+use serde_json::Value as SerdeValue;
+use std::sync::Arc;
+
+pub fn transform_serde_value<'a>(
```

Comment:
"transform" implies to me that we're mutating the object. I would still use the word "parse", so `parse_serde_value`. Same below.